### DISCUSSION TITLE
代理添加关闭功能

### DISCUSSION THREAD

**Discussion by tao917922822** [nikkinikki-org/OpenWrt-nikki#332] [Ideas]
![Screenshot_2024-12-08-13-15-55-497_mark via](https://github.com/user-attachments/assets/328fa875-23c3-4f09-90b2-31acc9668bf9)
能不能添加一个开关  有些人可能不需要udp的代理  谢谢

**Comment by morytyann** [comment]
暂时不会加，目前你可以在配置文件里加一条规则`NETWORK,udp,DIRECT`，或者在`代理配置 -> 绕过 -> 要代理的UDP目标端口`填个0也可以。
